lib/kino/input.ex

```diff
@@ -56,9 +56,16 @@ defmodule Kino.Input do
   @spec text(String.t(), keyword()) :: t()
   def text(label, opts \\ []) when is_binary(label) and is_list(opts) do
     default = Keyword.get(opts, :default, "")
+    assert_default_value!(default, "be a string", &is_binary/1)
```

Comment:
@josevalim let me know if you'd convert instead, but I'm thinking being explicit makes sense.

Reply:
I see, let's do that!